experimental/framework/presence/package.json

```diff
@@ -0,0 +1,105 @@
+{
+	"name": "@fluid-experimental/presence",
+	"version": "2.3.0",
+	"private": true,
+	"description": "A component for lightweight data sharing within a single session",
+	"homepage": "https://fluidframework.com",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/microsoft/FluidFramework.git",
+		"directory": "experimental/framework/presence"
+	},
+	"license": "MIT",
+	"author": "Microsoft and contributors",
+	"sideEffects": false,
+	"type": "module",
+	"exports": {
+		".": {
+			"import": {
+				"types": "./lib/index.d.ts",
+				"default": "./lib/index.js"
+			},
+			"require": {
+				"types": "./dist/index.d.ts",
+				"default": "./dist/index.js"
+			}
+		},
+		"./internal": {
+			"import": {
+				"types": "./lib/index.d.ts",
+				"default": "./lib/index.js"
+			},
+			"require": {
+				"types": "./dist/index.d.ts",
+				"default": "./dist/index.js"
+			}
+		}
+	},
+	"files": [
+		"dist/**/*",
+		"lib/**/*",
+		"*.d.ts",
+		"!**/test/**"
+	],
```

Comment:
What is "files" doing here?  We don't have this in other packages?

Reply:
We do have it in some - I see at least packages/framework/data-object-base/package.json and packages/tools/changelog-generator-wrapper/package.json.
The description "The 'files' field is an array of files to include in your project. If you name a folder in the array, then it will also include the files inside that folder." [docs](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#files)
So, this is an explicit list of what should be in the package (well it doesn't list `package.json`, `LICENSE`, and `README` that are always included) and is better than an ignore list (.npmignore).
We should probably have this in all of our packages.